content/docs/tech-insights/track-sonarqube/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Track On-prem Sonarqube metrics
+publishedDate: '2023-08-04T15:15:00.0Z'
+description: How to track and visualize a Sonarqube metrics with Roadie
+---
+
+One of the most advantageous steps a platform team can take to support application developers (app devs) is to offer them a reliable and secure platform for enhancing code quality. This is often achieved by incorporating SonarCloud into the development process, enabling teams to identify and rectify code-related issues effectively.
+
+The [Sonarcloud tutorial](/docs/tech-insights/track-sonarcloud) explains how to configure Roadie to use the publicly accessible API of Sonarcloud. This tutorial instead assumes that you are self-hosting Sonarqube and the API is within a private network. In this tutorial we will be using the Roadie broker service to make the connection between Sonarqube and Roadie.
+
+By tracking issues and hotspots reported by Sonarqube across your organization allows you to highlight and to proactively address potential pitfalls, enhance code integrity, and bolster application security. This practice not only fosters a culture of continuous improvement but also ensures that software projects evolve with a heightened focus on robustness and reliability.
+
+With Roadie’s Tech Insights feature, you can monitor data from services within your private network right within your software catalog:
+
+1. Automatically scan Components in the Backstage catalog and record the number of issues and hotspots reported by Sonarqube
+2. Visualise the projects which are doing well, and also the worst offenders.
```

Comment:
This is not part of this tutorial any more.

Reply:
done.